### ISSUE TITLE
Add support in Tracer for additional arguments in the lambda_handler

### ISSUE THREAD

**Issue by hanninen** [awslabs/aws-lambda-powertools-python#242]
**Is your feature request related to a problem? Please describe.**
We have a decorator that injects a new argument to the lambda_handler and this is not compatible with the current implementation of the `Tracer.capture_lambda_handler`, as it calls the passed `lambda_handler` with the standar `event` and `context` arguments only.

**Describe the solution you'd like**
Allow other decorators to add new arguments to the `lambda_handler` and pass `**kwargs` in addition to the `event` and `context` variables to the `lambda_handler` call.

**Describe alternatives you've considered**
Can the `Tracer.capture_lambda_handler` be changed not to call the `lambda_handler`, but pass down the decorated function with the `tracer` instance?

**Comment by heitorlessa** [comment]
Hi @hanninen, thanks for raising this - We're returning from PTO this week.

Do you mean like this? 

```python
...
@tracer.capture_lambda_handler
def handler(event, context, **kwargs):
    return dummy_response
```

If so, sure thing, it'd be a no brainier for us to ship it in the next release - let us know!

Here's a test to exemplifies this better:

```python
def test_capture_lambda_handler_with_additional_kwargs(dummy_response):
    # GIVEN tracer lambda handler decorator is used
    tracer = Tracer(disabled=True)

    # WHEN a lambda handler signature has additional keyword arguments
    @tracer.capture_lambda_handler
    def handler(event, context, my_extra_option=None, **kwargs):
        return dummy_response

    # THEN tracer should not raise an Exception
    handler({}, {}, blah="blah")
```

Thank you

**Comment by hanninen** [comment]
Yes `**kwargs` should do it just fine, thanks!

**Comment by heitorlessa** [comment]
Thanks @hanninen - It'll be available tomorrow in the 1.10 release

**Comment by heitorlessa** [comment]
hey @hanninen - **This is now available on 1.10 on PyPi and on SAR** :) 

Closing this now, and thank you for raising it
